Client.Linq.Test/InfluxDBQueryVisitorTest.cs

```diff
@@ -197,9 +199,9 @@ public void ResultOperatorTakeSkipMultipleTimes()
                 Assert.AreEqual(expected, visitor.BuildFluxQuery(),
                     $"Expected Flux: {flux} for expression {expression}.");
 
-                foreach (var (index, value) in assignments)
-                    Assert.AreEqual(value, GetLiteral<IntegerLiteral>(ast, index).Value,
-                        $"Expected: {value} with index: {index} for Queryable expression: {queryable}");
+                foreach (var kvp in assignments)
```

Comment:
From my perspective, the `(key, value)` syntax seems preferable. Is there a specific reason to opt for the `var kvp` instead?

```suggestion
                foreach (var (key, value) in assignments)
```

Reply:
The only reason for this change is to support .NET Framework 4.8, because [KeyValuePair<TKey,TValue>.Deconstruct()](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.keyvaluepair-2.deconstruct) is also only available starting with .NET Core 2.0.
I think this change only affects code readability a little bit, but not performance.

Reply:
It makes sense. Thanks for explanations.